sdk/python/kfp/v2/dsl/artifact_test.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for kfp.v2.ds.artifact module."""
+import importlib
+import mock
+import unittest
+import textwrap
+
+from kfp.v2.dsl import artifact
+
+
+class _MyArtifact(artifact.Artifact):
+  TYPE_NAME = 'MyTypeName'
+  PROPERTIES = {
+      'int1': artifact.Property(
+          type=artifact.PropertyType.INT,
+          description='An integer-typed property'),
+      'int2': artifact.Property(type=artifact.PropertyType.INT),
+      'float1': artifact.Property(
+          type=artifact.PropertyType.DOUBLE,
+          description='A float-typed property'),
+      'float2': artifact.Property(type=artifact.PropertyType.DOUBLE),
+      'string1': artifact.Property(
+          type=artifact.PropertyType.STRING,
+          description='A string-typed property'),
+      'string2': artifact.Property(type=artifact.PropertyType.STRING),
+  }
+
+
+_SERIALIZED_INSTANCE = """\
+{
+  "properties": {
+    "float1": {
+      "doubleValue": 1.11
+    },
+    "int1": {
+      "intValue": "1"
```

Comment:
re: `"1"`
Is quoting optional? Don't see quotes on `1.11`.

Reply:
No. It's interesting but I think double quotes are required here.